flutter/test/load_contexts_integrations_test.dart

```diff
@@ -74,23 +72,22 @@ void main() {
     expect(fixture.options.eventProcessors.length, 1);
 
     final e = SentryEvent();
-    final event = await (fixture.options.eventProcessors.first(e)
-        as FutureOr<SentryEvent>);
+    final event = await fixture.options.eventProcessors.first.apply(e);
 
     expect(called, true);
-    expect(event.contexts.device!.name, 'Device1');
-    expect(event.contexts.app!.name, 'test-app');
-    expect(event.contexts.operatingSystem!.name, 'os1');
-    expect(event.contexts.gpu!.name, 'gpu1');
-    expect(event.contexts.browser!.name, 'browser1');
+    expect(event?.contexts.device!.name, 'Device1');
```

Comment:
would it make sense to be `!` instead of `?` here and all the following tests?

Reply:
I'm wondering the same but I think having `?` is slightly better because you'll get the error or failure message from the `excpect` instead of Dart runtime error message.

Reply:
makes sense, but then it should be `?` everywhere, right, if `device` is null and using `!` wont work either